contentcuration/contentcuration/frontend/shared/views/MarkdownEditor/MarkdownEditor/MarkdownEditor.vue

```diff
@@ -159,6 +159,19 @@
           this.initImageFields();
         }
       },
+      'file.error'() {
+        // eslint-disable-next-line
+        console.error('The image could not be uploaded');
+      },
+      'file.progress'(progress) {
+        if (progress === 0) {
+          // eslint-disable-next-line
+          console.log('The image upload has started');
```

Comment:
Probably good to remove this later, but makes sense to have it in the short term